packages/dashboard/src/customization/propertiesSections/settings/section.test.tsx

```diff
@@ -26,5 +27,16 @@ it('renders', () => {
 
 it('renders the decimal places input', async () => {
   const elem = render(<TestComponent />).baseElement;
-  expect(getByLabelText(elem, 'decimal places')).toBeTruthy();
+  expect(getByLabelText(elem, 'Places')).toBeTruthy();
+});
+
+it('shows error if decimal places input is greater than 100', async () => {
+  const user = userEvent.setup();
+  const { getByTestId, getByText } = render(<TestComponent />);
+  const inputElem = getByTestId('decimal-place-config').querySelector(
+    'input'
+  ) as HTMLInputElement;
```

Comment:
Can we update this test to use `screen` to select and use an accessible property, such as the label or placeholder text, instead of a test ID?

Reply:
Done